app/components/crate-header.hbs

```diff
@@ -3,6 +3,13 @@
     <span data-test-crate-name>{{@crate.name}}</span>
     {{#if @version}}
       <small data-test-crate-version>v{{@version.num}}</small>
+
+      {{#if @version.yanked}}
+        <span local-class="yanked-badge">
+          {{svg-jar "trash"}}
+          Yanked
+        </span>
```

Comment:
Do we want to link to something — or at least have a tooltip — which explains what yanking means in the same way the previous version said "This crate has been yanked, but it is still available for download for other crates that may be depending on it"?

Reply:
that sounds like a good idea, thanks! 👍

Reply:
![Bildschirmfoto 2023-05-03 um 16 27 14](https://user-images.githubusercontent.com/141300/235946134-b8f9afe1-745e-4161-9600-52dfbf20978d.png)